src/VMware/examples/Lock-AzVMwareVirtualMachineMovement.md

```diff
@@ -0,0 +1,18 @@
+### Example 1: {{ Add title here }}
+```powershell
+PS C:\> {{ Add code here }}
+
+{{ Add output here }}
```

Comment:
Please fill placeholder.

Reply:
The command is invalid and has been hidden